src/services/git/GitHubService.ts

```diff
@@ -442,41 +424,41 @@ export class GitHubService implements IVCSService {
       body: val.body,
       createdAt: val.created_at,
       updatedAt: val.updated_at,
-      authorAssociation: val.author_association,
+      authorAssociation: val.user.login,
```

Comment:
Delete? We already store `val.user.login` for user.login on line 420. I don't think the user will always be the one who is associated to the issue.

Reply:
no, becuase interfaces requires the `authorAssociation` and it can have a deeper meaning here